webapp/src/Service/ImportExportService.php

```diff
@@ -456,16 +457,7 @@ public function getTeamData(): array
     /**
      * Get results data for the given sortorder.
      *
```

Comment:
```suggestion
    /**
```

The only relevant bit is the sortorder, so I suggest we remove it.